packages/animations/example/lib/container_transition.dart

```diff
@@ -200,10 +203,16 @@ class _OpenContainerTransformDemoState
           ),
           const SizedBox(height: 16.0),
           ...List<Widget>.generate(10, (int index) {
-            return OpenContainer(
+            return OpenContainer<bool>(
               transitionType: _transitionType,
               openBuilder: (BuildContext _, VoidCallback openContainer) {
-                return _DetailsPage();
+                return const _DetailsPage(includeMarkAsDoneBtn: true);
+              },
+              onClosed: (bool isMarkedAsDone) {
+                if (isMarkedAsDone) {
+                  scaffoldKey.currentState.showSnackBar(
+                      const SnackBar(content: Text('Marked as done!')));
```

Comment:
nit: use a trailing comma for better formatting.

```suggestion
                  scaffoldKey.currentState.showSnackBar(
                    const SnackBar(content: Text('Marked as done!'),
                  ));
```